.github/workflows/pull-request.yml

```
@@ -37,6 +37,29 @@ jobs:
           echo "Error: Source and Target Branches are the same. Please ensure they are different."
           echo "Error: Close this PR and try again."
           exit 1
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)